### PR TITLE
Demo docs for Sbt tool configuration

### PR DESCRIPTION
**Pull request by aduprat**
I had problems while configure Sbt, so I implemented an integration test.

This integration test may be useful, as it's presenting an example of how to configure Sbt.

### CODE REVIEW

File: integrations/src/test/resources/io/jenkins/plugins/casc/SbtConfiguratorTest.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+tool:
```
Comment (timja) [inline integrations/src/test/resources/io/jenkins/plugins/casc/SbtConfiguratorTest.yml]: Ideally this test would go to the plugin, and then documented on the plugin docs and a demo added here

**Comment by casz** [comment]
I suggest you move this test to the actual plugin.

**Comment by casz** [comment]
@aduprat not the plugin module in this repo. but add the test to the sbt plugin repo: https://github.com/jenkinsci/sbt-plugin

**Comment by aduprat** [comment]
> @aduprat not the plugin module in this repo. but add the test to the sbt plugin repo: https://github.com/jenkinsci/sbt-plugin

Sorry, I misunderstood...

**Comment by aduprat** [comment]
Do you still want the sample documentation?

**Comment by casz** [comment]
Ya of course @aduprat 😄

**Comment by casz** [comment]
I will be good to get sbt-plugin updated to Java 8 and modernized so it will be able to run on Java 11 runtime in the near future.

**Pull request merged**